Enable .NET Framework F5 & VS TestExplorer support for libraries

ViktorHofer opened the pull request:
Tried it out locally, this works now :)

cc @Anipik

ViktorHofer committed:
Enable .NET Framework F5 & TestExplorer

Adding a launchSettings profile to enable .NET Framework F5 scenarios.
Also setting the DEVPATH variable for the -vs hook to use the locally
built .NET Framework runtime.

safern commented on eng/build.ps1:
```diff
@@ -85,6 +85,7 @@ if ($vs) {
 
     # This tells .NET Core to use the same dotnet.exe that build scripts use
     $env:DOTNET_ROOT="$PSScriptRoot\..\artifacts\bin\testhost\netcoreapp5.0-Windows_NT-$configuration-$archTestHost";
+    $env:DEVPATH="$PSScriptRoot\..\artifacts\bin\testhost\net472-Windows_NT-$configuration-$archTestHost";
```
Is DEVPATH just a thing of Full Framework? If not should we condition it out for netcoreapp and other tfms?

ViktorHofer replied:
Yeah that's .NET Framework only.

ViktorHofer committed:
Set DEVPATH env in F5

ViktorHofer commented:
I just added the DEVPATH environment variable to the launchSettings.json as well (we had that setting in Arcade before but at some point I removed the .NET Framework profile as it was never working). This means that .NET Framework F5 now works without the -vs switch :)

ViktorHofer commented:
Given that both changes aren't exercised in CI, I will merge this :)
